src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -971,7 +971,7 @@ export function getAlertContent(
     blocks.push(
       <p key="you-applied-on">
         <strong>You applied on:</strong>{' '}
-        {moment(applicationDate).format('MMMM D, YYYY')}
+        {format(Date.parse(applicationDate), 'MMMM d, yyyy')}
```

Comment:
This is in an untested function.  The only place it is used is in `src/applications/personalization/dashboard/components/HCAStatusAlert.jsx`